Packs/Kafka/Integrations/KafkaV3/KafkaV3.py

```diff
@@ -0,0 +1,633 @@
+    else:
+        partition = None  # type: ignore
+
+    kafka.produce(
+        value=str(value),
+        topic=str(topic),
+        partition=partition  # type: ignore
+    )
+
+
+def consume_message(kafka: KafkaCommunicator, demisto_args: dict) -> Union[CommandResults, str]:
+    """Consume one message from topic
+
+    Args:
+        kafka: initialized KafkaCommunicator object to preform actions with.
+        demisto_args: The demisto command arguments.
+
+    Return CommandResults with the relevant message from kafka.
+    """
+    topic = str(demisto_args.get('topic'))
+    partition = int(demisto_args.get('partition', -1))
+    offset = demisto_args.get('offset', '0')
+
+    message = kafka.consume(topic=topic, partition=partition, offset=offset)
+    demisto.debug(f"got message {message} from kafka")
+    if not message:
+        return 'No message was consumed.'
+    else:
+        message_value = message.value()
+        dict_for_debug = [{'Offset': message.offset(), 'Message': message_value.decode('utf-8')}]
+        demisto.debug(f"The dict for debug: {dict_for_debug}")
+        message_value = message.value()
+        readable_output = tableToMarkdown(f'Message consumed from topic {topic}',
+                                          [{'Offset': message.offset(), 'Message': message_value.decode("utf-8")}])
+        content = {
+            'Name': topic,
+            'Message': {
+                'Value': message_value.decode('utf-8'),
+                'Offset': message.offset()
+            }
+        }
+
+        return CommandResults(
+            outputs=content,
+            readable_output=readable_output,
+            outputs_key_field='Name',
+            outputs_prefix='Kafka.Topic'
+        )
+
+
+def fetch_partitions(kafka: KafkaCommunicator, demisto_args: dict) -> CommandResults:
+    """Get available partitions in a given topic
+
+    Args:
+        kafka: initialized KafkaCommunicator object to preform actions with.
+        demisto_args: The demisto command arguments.
+
+    Return CommandResults with the relevant partitions.
+    """
+    topic = demisto_args.get('topic')
+    kafka_topics = kafka.get_topics()
+    if topic in kafka_topics:
+        kafka_topic = kafka_topics[topic]
+        partition_objects = kafka_topic.partitions.values()
+        partitions = [partition.id for partition in partition_objects]
+
+        readable_output = tableToMarkdown(
+            name='Available partitions for topic \'{}\''.format(topic),
+            t=partitions,
+            headers='Partitions'
+        )
+        return CommandResults(outputs_prefix='Kafka.Topic',
+                              outputs_key_field='Name',
+                              outputs={'Name': topic, 'Partition': partitions},
+                              readable_output=readable_output)
+    else:
+        raise DemistoException(f'Topic {topic} was not found in Kafka')
+
+
+def handle_empty(value: Any, default_value: Any) -> Any:
```

Comment:
i dont really understand why we need that func and not just use `.get()` ?

Reply:
In case the demisto.param is present but empty like {'some_param': ''}